tests/benchmarks/test_bench_ss.py

```
@@ -0,0 +1,59 @@
+@pytest.fixture
+def ss(ardos):
+    return ardos(md=True, ss=True)
+
+
+def test_create_and_delete_churn(ss, channel_conn, benchmark):
+    sender = channel_conn()
+    counter = [1_500_000]
+    cls = class_id("test.dc", "DistributedTestObject1")
+
+    def step():
+        do_id = counter[0]; counter[0] += 1
```

Ruff (0.15.11)

[error] Multiple statements on one line (semicolon)

(E702)